compiler/daml-lf-verify/tests/DA/Daml/LF/Verify/Tests.hs

```diff
@@ -16,10 +16,11 @@ import DA.Bazel.Runfiles
 
 mainTest :: IO ()
 mainTest = defaultMain $ testGroup "DA.Daml.LF.Verify"
-  [ quickstartTests
-  , conditionalTests
+  [ -- quickstartTests, -- TODO: reinstate
```

Comment:
Since I don’t like having my stamp on things I don’t understand: I assume you’ve agreed on this with @hurryabit and @Gertjan423?

Reply:
This is only very temporary. The test fails due to a bug on my part. I'm currently working on understanding and fixing the bug. This should certainly be fixed by the weekend :)

Reply:
Thanks for confirming!

Reply:
Yes, sorry for the surprise. Gertjan & I talked about this and decided it would be best to just disable it temporarily.

Reply:
See PR#6431.